firmware/console/binary_log/log_field.cpp

```diff
@@ -43,10 +43,9 @@ void LogField::writeHeader(Writer& outBuffer) const {
 	buffer[54] = m_digits;
 
 	// Offset 55, (optional) category string
+	memset(&buffer[55], 0, 34);
```

Comment:
we have `efi::clear`, maybe useful

Reply:
I poked at this but the generics confused me: does this support `char[]` yet?